src/middlewared/middlewared/plugins/jail.py

```diff
@@ -1501,7 +1503,7 @@ class JailFSAttachmentDelegate(FSAttachmentDelegate):
         except Exception:
             pass
         else:
-            if query_dataset.startswith(os.path.join(activated_pool, 'iocage')):
+            if query_dataset.startswith(os.path.join(activated_pool or '', 'iocage')):
```

Comment:
This won't work correctly:
```
>>> os.path.join('', 'iocage')
'iocage'
```

Reply:
Yes, it won't. Thank you for catching that